share/isso.conf

```diff
@@ -112,6 +112,13 @@ reload = off
 # in production.
 profile = off
 
+# an optional list of reverse proxies IPs behind which you have deployed
+# your Isso web service (e.g. `127.0.0.1`).
+# This allow for proper remote address resolution based on a
+# `X-Forwarded-For` HTTP header, which is important for the mechanism
+# forbiding several comment votes coming from the same IP.
```

Comment:
s/forbiding/forbidding/

I think this should say subnet rather than IP since it calls utils.anonimize().

Reply:
👍